docs/1-when-the-music-starts/4-inner-data-science-loop.md

```diff
@@ -61,30 +69,28 @@ You can see information like where the model is stored, version, details about m
 
 ## Model Serving
 
-Now that we have our model artifacts saved in a bucket, we can deploy it in our data science project. The beauty of OpenShift AI, along with the underlying KServe technology, is that we don’t have to worry about the containerization of the model or the runtime. It abstracts away these complexities. All we have to do is select the right runtime for our model and point where the model is. 
+Now that we have our model artifacts saved in a bucket, we can deploy it in our data science project. The beauty of OpenShift AI, along with the underlying KServe technology, is that we don’t have to worry about the containerization of the model or the runtime. It abstracts away these complexities. All we have to do is select the right runtime for our model and point where the model is.
+
 Let's give it a try:
 
-1. Go to your Data Science project > `<USER_NAME>` > `Models` > Select `Single-Model Serving Platform`. Then `Deploy 
+1. Go to Model Registry and find the model you just registered. Click `Actions` > `Deploy`.
+   
+    ![deploy-from-registry.png](./images/deploy-from-registry.png)
+
 
-    ![single-model-serving.png](./images/single-model-serving.png)
+2. Select `<USER_NAME>-jukebox` as your target project to deploy the model.
 
-    ![single-model-serving-2.png](./images/single-model-serving-2.png)
+    ![deploy-from-registry-2.png](./images/deploy-from-registry-2.png)
 
-2. Fill out the form by the following information:
+3. Update the form with the following information:
 
 - Model deployment name: `jukebox`
 - Serving runtime: `OpenVino Model Server`
 - Model framework (name - version): `onnx - 1`
 - Model server replicas: `1`
 - Model server size: `Small`
-- Model route:
-    -  Select `Make deployed models available through an external route`
```

Comment:
We no longer call this out in text?

Reply:
oh very good catch..must have removed by mistake 🤦